### PR TITLE
[DataObject] Error Handling for wrong formatted value for date datatypes

### LINKED ISSUE #9088: Error Handling for wrong formatted Customer data
If one is creating (or updating an existing) customer and fills in a wrong formatted birth date:
CMF_Fehler

After clicking Save button the input field will be **red framed** but toolbox tells **"Saved successfully!"** and reloading of the Customer dismisses all "updated" values.
There should be an alert message instead.

### PR DESCRIPTION
**Pull request by weisswurstkanone**
resolves #9088

### CODE REVIEW

**Comment by dvesh3** [comment]
@weisswurstkanone please rebase it on 10.0. thanks!

**Comment by weisswurstkanone** [comment]
done

**Review by dvesh3 — CHANGES_REQUESTED** [review]
image

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/datetime.js [hunk 1]
```diff
@@ -132,8 +132,14 @@ pimcore.object.tags.datetime = Class.create(pimcore.object.tags.abstract, {
                 dateString += " 00:00";
             }
 
-            var date = Ext.Date.parseDate(dateString, "Y-m-d H:i").getTime();
-            return date;
+            let value = Ext.Date.parseDate(dateString, "Y-m-d H:i")
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/datetime.js]:
```suggestion
            value = Ext.Date.parseDate(dateString, "Y-m-d H:i");
```

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/datetime.js [hunk 2]
```diff
@@ -132,8 +132,14 @@ pimcore.object.tags.datetime = Class.create(pimcore.object.tags.abstract, {
...
+            if (typeof value.getTime == "function") {
+                return value.getTime();
+            } else {
+                return value;
+            }
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/datetime.js]:
```suggestion
            if (typeof value.getTime == "function") {
                return value.getTime();
            }
```

**Pull request merged**